lib/core/form/components/widgets/core/form-field.model.ts

```diff
@@ -134,6 +135,20 @@ export class FormFieldModel extends FormWidgetModel {
         return this._isValid;
     }
 
+    isTabVisible() {
```

Comment:
The visibility doesn't look like a concern of this model what di you think @DenysVuika @mauriziovitale ?

Reply:
Misses the return value typing. Besides that, the form model already parses tabs and tab fields on init, so I think we are already bound with the notion of tabs. I would suggest leaving here and maybe thinking over splitting in the future

Reply:
but not visibility

Reply:
@DenysVuika should it be concern of visibility service?

Reply:
@eromano probably yes. If that api moves to the visibility service, it should also take `findFieldByCurrentId` function there too

Reply:
Yep, I agree it should not be there.

Reply:
I see that we already do some checks on tab visibility 
```
  public refreshVisibility(form: FormModel) {
        if (form && form.tabs && form.tabs.length > 0) {
            form.tabs.map((tabModel) => this.refreshEntityVisibility(tabModel));
        }
```

Reply:
The issue it's related to form completion. 
Right now we cannot complete a task that has an invisible tab with an invalid form inside.
If we have just an invisible invalid field we can complete it.
In this PR I added a check to make the field invisible if the tab that belongs to is also invisible.